adapters/nanoleaf/NL08_0800.py

```diff
@@ -0,0 +1,29 @@
+import json
+from adapters.base_adapter import Adapter
+from devices.dim_light import DimLight
+
+class NL08_0800(Adapter):
+    """adapter for the Nanoleaf Smart Ivy bulbs"""
+    def __init__(self, devices):
+        super().__init__(devices)
+        self.devices.append(DimLight(devices, 'dimmer', 'state_brightness'))
+        
+    def handleMqttMessage(self, device_data, message):
```

Comment:
You could just remove this method as we have the same in Adapter which this class is inherited from